src/Rector/v10/v0/ChangeDefaultCachingFrameworkNamesRector.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ssch\TYPO3Rector\Rector\v10\v0;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+use TYPO3\CMS\Core\Core\CacheManager;
```

Comment:
This is wrong. Change Core with Cache and it should work

Reply:
I overlooked that.